css/form.css

```diff
@@ -427,6 +449,14 @@ html[dir="rtl"] .radio.radio-icon input[type="radio"] + label > span.check {
   background-color: #337ab7;
 }
 
+.radio.radio-icon.readonly input[type="radio"]:checked + label > span.check {
+  background-color: #e1e1e1;
+}
+
+.radio.radio-icon.readonly input[type="radio"] + label > span.check{
```

Comment:
Hi @tonytlwu 

I did npm install on master branch, but when I created new branch from master for XOR-156, Code formatter stopped working and surprisingly it was not shown in my editor also. So After your comments I again did npm i for newly created branch XOR-156, so now code formatter is working fine. I think similar thread was discussed [here](https://flipletoutsourced.slack.com/archives/C0383397GN4/p1663664936975419) 

So to prevent this error again will do npm i for each branch and informed everyone in team also.